server/matchmaker/search.py

```diff
@@ -40,8 +42,29 @@ def __init__(self, player, start_time=None, rating_prop='ladder_rating'):
             0: 0.8
         }
 
+    @property
```

Comment:
Why do we need property, not function, historic reasons?

Properties by default are evil